Sources/SwiftUICharts/Base/Grid/ChartGridShape.swift

```diff
@@ -0,0 +1,30 @@
+import SwiftUI
+
+struct ChartGridShape: Shape {
+    var numberOfHorizontalLines: Int
+    var numberOfVerticalLines: Int
+
+    func path(in rect: CGRect) -> Path {
+        let path = Path.drawGridLines(numberOfHorizontalLines: numberOfHorizontalLines,
+                                      numberOfVerticalLines: numberOfVerticalLines,
+                                      in: rect)
+        return path
+    }
+}
+
+struct ChartGridShape_Previews: PreviewProvider {
+    static var previews: some View {
+        Group {
+            ChartGridShape(numberOfHorizontalLines: 5, numberOfVerticalLines: 0)
+                .stroke()
+                .rotationEffect(.degrees(180), anchor: .center)
```

Comment:
It also seems that these rotations are not necessary here. When I comment them the preview stays the same. But maybe I am just missing something important.

Reply:
Ok, I think I got it. Are you trying to get `x` and `y` axis to be the same as you would intuitively expect? As in, point `(0.0)` is not top left corner but you want it to be lower left?

I think padding of line `.rotation3DEffect(.degrees(180), axis: (x: 0, y: 1, z: 0))` is not correct.

Reply:
done